src/lib/components/stackedbarchart.svelte

```diff
@@ -57,28 +58,34 @@
 		<g bind:this={gy} transform="translate({padding}, {0})" />
 		<g>
 			{#each index as [date, values]}
-				<g
-					class="stacked-bar"
-					role="none"
-					style="margin: 0; padding; 0; gap: 0;"
-					on:mouseenter={(d) => handleMouseOver(d, { date, values })}
-					on:mouseleave={handleMouseOut}
+				<a
+					href={`posts?${category ? `category=${category}` : ''}				
+				${date ? `&start=${date}` : ''}
+				${date ? `&end=${date}` : ''}`}
```

Comment:
A note that, by default, SvelteKit prefetches data and code for future pages on hover: <https://kit.svelte.dev/docs/link-options>. You may or may not want to change that behavior for this chart.

Reply:
I think this series of links in particular would benefit from changing the hover behavior because there are so many in a single place, and the changes that a person clicks on one are low. But for other links on the page, there is a higher chance the hovering will result in a click.